cmd/kubectl-gadget/utils/errors.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2019-2021 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"errors"
+	"fmt"
+)
+
+// Gadget pod
+var (
+	ErrGadgetPodNotFound      = errors.New("gadget pod not found")
+	ErrMultipleGadgetPodFound = errors.New("multiple gadget pods found")
+)
+
+// Kubernetes client
+func WrapInErrSetupK8sClient(err error) error {
```

Comment:
Are all of these errors used at least twice?

Reply:
Yeah, it was also my criteria to create them. Except for three of them but I think it is very probably that will be reused in the future. For instance, `WrapInErrGenGadgetOutput` (will probably be used by networkpolicy-advisor) and other like `WrapInErrParamsNotSupported` and `WrapInErrInvalidArg` are currently being used just once.